Formula/texmaker.rb

```diff
@@ -5,15 +5,14 @@ class Texmaker < Formula
   sha256 "526896f2c1ae561130eec7aae815b9dcda9e8eeb772b6541d0dc94ce91a71044"
   license "GPL-2.0-only"
 
+  depends_on xcode: :build
   depends_on "qt"
 
-  depends_on xcode: :build
   on_linux do
     depends_on "gcc"
+    fails_with gcc: "5"
```

Comment:
Isn't this failure generic? I don't think you need to keep this in the `on_linux` block.

Reply:
Ok. I am anyway not sure on this. I found some comment in another Qt-based formula which mentioned that GCC 5 causes problems. But doesn't e.g. Mac anyway uses Clang instead?